### PR TITLE
Add explicit tests for byte ordering in Guid

### LINKED ISSUE #6315: new Guid(byte[]) should explicitly test byte ordering
The only tests for the byte array ctor for Guid use Guid.ToByteArray and round-trip the value.  While this is good (proving we read what we write), there is an order to that input byte array, and we should verify it against a static array / static GUID.

### PR DESCRIPTION
**Pull request by hughbe**
- Adds new tests for constructors taking byte arrays
- Fixes #6315

/cc @bartonjs

### CODE REVIEW

**Comment by bartonjs** [comment]
Ah, that "lovely" multi-little-endian byte ordering that just makes you scratch your head and blink.

LGTM, and thanks!

**Comment by hughbe** [comment]
Yeah I was pretty confused at first!! Haha

File: src/System.Runtime/tests/System/Guid.cs [hunk 1]
```diff
@@ -5,7 +5,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-
```
Comment (JonHanna) [inline src/System.Runtime/tests/System/Guid.cs]: Just a nit, but the space after the system usings is preferred, unless I missed a change to that.

**Comment by JonHanna** [comment]
Good addition.

File: src/System.Runtime/tests/System/Guid.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Runtime/tests/System/Guid.cs]: Yeah The coding guidelines were recently changed

**Comment by hughbe** [comment]
@bartonjs it was your suggestion for this test, so thanks. are there any similar additions you feel are necessary for tests for Guid or other classes? I can submit some more tests if u have any examples

**Pull request merged**